content/news/XXXX-XX-XX-new_icon_contest.md

```diff
@@ -0,0 +1,76 @@
+
+We also alter the Mixxx Logo for seasonal events like Christmas:
+
+![Mixxx Icon Logo Christmas Original Proposal]({static}/images/news/icon-contest/mixxx-icon-logo-christmas_orig_proposal.svg)
+![Mixxx Icon Logo Christmas]({static}/images/news/icon-contest/mixxx-icon-logo-christmas.svg)
+
+## Examples of other DJ software
+The following links show Google image search results for the use of icons by competing DJ software. These are just examples to show the variety of styles and combinations of text Logo and square Icon. The new updated Mixxx Icon should allow for a similar variety of styles and combinations, while it should be distinctive to be recognized as Mixxx.
+
+- [Algoridm DJay Pro](https://www.google.com/search?client=firefox-b-d&q=Algoriddim+DJay+Pro+Label+Logo+Icon&udm=2)
+- [Serato DJ Pro](https://www.google.com/search?client=firefox-b-d&q=Serato+DJ+Pro+Label+Logo+Icon&udm=2)
+- [Virtual DJ](https://www.google.com/search?client=firefox-b-d&udm=2&q=Atomix+VirtualDJ+Label+Logo+Icon&spell=1&dpr=1)
+- [Traktor Pro](https://www.google.com/search?q=Native+Instruments+Traktor+Pro+Label+Logo+Icon&client=firefox-b-d&udm=2&uact=5)
+- [PioneerDJ Rekordbox](https://www.google.com/search?client=firefox-b-d&q=Pioneer+DJ+Rekordbox+Label+Logo+Icon&udm=2)
+- [DenonDJ Engine Prime](https://www.google.com/search?client=firefox-b-d&q=Denon+DJ+Engine+Prime+OS+Label+Logo+Icon&udm=2)
+
+DJ Studio has even a dedicated subpage with variations of it's Logo and Icon for the use by the DJ Press, explaining the different uses: [DJ Studio MediaKit](https://dj.studio/media-kit)
+
+## Platform style guidelines to consider
+The following links show the style guidelines for app icons of the most common platforms. It's not part of the contest to provide icons for all platforms, but the design should allow all these adaptations, without loosing it's distinctive character.
+
+- [Apple](https://developer.apple.com/design/human-interface-guidelines/macos/icons-and-images/app-icon/)
+- [Microsoft](https://learn.microsoft.com/en-us/windows/apps/design/style/iconography/app-icon-design)
+- [Flathub](https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines/quality-guidelines/#app-icon)
+- [Google](https://developer.android.com/distribute/google-play/resources/icon-design-specifications)
+- [KDE](https://develop.kde.org/hig/icons/colorful/application/)
+- [Gnome](https://developer.gnome.org/hig/guidelines/app-icons.html)
+
+## Submission guidelines
+### License and ownership of the design
+The Mixxx brand, including logo and icon, is not licensed under the GPL like the software itself. The design is the property of the Mixxx e.V., therefore, it is necessary to sign the Mixxx Contributor Agreement and submit the contest contribution under the terms of this agreement. The Mixxx brand is hold by Mixxx e.V. and used to distinguish the official version of the software provided by the Mixxx development team from forks with altered functionality/quality.
```

Comment:
We need to file a separate copyright agreement that not mentions GPL and grands all rights exclusively and not revocable to the e.V.